rules/sat/sat-inscript2.js

```diff
@@ -3,103 +3,60 @@
 
 	var satInScript2 = {
 		id: 'sat-inscript2',
-		name: 'संताली इनस्क्रिप्ट २',
+		name: 'inskript2',
```

Comment:
This can be "inscript2" written in Ol Chiki. If it is in English, note the spelling mistake